From 5fb7d456be62776541d61107f28c06b79c6a6544 Mon Sep 17 00:00:00 2001
From: Honggang Li <honli@redhat.com>
Date: Thu, 19 Oct 2017 18:34:01 +0800
Subject: [PATCH] Add shebang for exp scripts

The shebang '#!/usr/bin/expect -f' will make opa-ff packages require
expect rpm package.

Following bash script had been applied to fix this issue.

for f in $(find -type f -name '*.exp') ; do
	sed -e '1 i\#!/usr/bin/expect -f' -i $f
done

Signed-off-by: Honggang Li <honli@redhat.com>
---
 IbaTools/FastFabric/chassis_configure.exp          | 1 +
 IbaTools/FastFabric/chassis_fmconfig.exp           | 1 +
 IbaTools/FastFabric/chassis_fmcontrol.exp          | 1 +
 IbaTools/FastFabric/chassis_fmgetconfig.exp        | 1 +
 IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp | 1 +
 IbaTools/FastFabric/chassis_fmsecurityfiles.exp    | 1 +
 IbaTools/FastFabric/chassis_getconfig.exp          | 1 +
 IbaTools/FastFabric/chassis_reboot.exp             | 1 +
 IbaTools/FastFabric/chassis_upgrade.exp            | 1 +
 IbaTools/FastFabric/configipoib.exp                | 1 +
 IbaTools/FastFabric/ff_function.exp                | 1 +
 IbaTools/FastFabric/ipoibping.exp                  | 1 +
 IbaTools/FastFabric/load.exp                       | 1 +
 IbaTools/FastFabric/mpiperf.exp                    | 1 +
 IbaTools/FastFabric/mpiperfdeviation.exp           | 1 +
 IbaTools/FastFabric/reboot.exp                     | 1 +
 IbaTools/FastFabric/sacache.exp                    | 1 +
 IbaTools/FastFabric/switch_capture.exp             | 1 +
 IbaTools/FastFabric/switch_configure.exp           | 1 +
 IbaTools/FastFabric/switch_dump.exp                | 1 +
 IbaTools/FastFabric/switch_fwverify.exp            | 1 +
 IbaTools/FastFabric/switch_getconfig.exp           | 1 +
 IbaTools/FastFabric/switch_hwvpd.exp               | 1 +
 IbaTools/FastFabric/switch_info.exp                | 1 +
 IbaTools/FastFabric/switch_ping.exp                | 1 +
 IbaTools/FastFabric/switch_reboot.exp              | 1 +
 IbaTools/FastFabric/switch_upgrade.exp             | 1 +
 IbaTools/FastFabric/upgrade.exp                    | 1 +
 TestTools/basic.exp                                | 1 +
 TestTools/chassis.exp                              | 1 +
 TestTools/common_funcs.exp                         | 1 +
 TestTools/defs.exp                                 | 1 +
 TestTools/extmng.exp                               | 1 +
 TestTools/ib.exp                                   | 1 +
 TestTools/ibtools.exp                              | 1 +
 TestTools/install.exp                              | 1 +
 TestTools/mpi.exp                                  | 1 +
 TestTools/network.exp                              | 1 +
 TestTools/opa_to_xml.exp                           | 1 +
 TestTools/proc_mgr.exp                             | 1 +
 TestTools/sm_control.exp                           | 1 +
 TestTools/switch_methods.exp                       | 1 +
 TestTools/target.exp                               | 1 +
 TestTools/tools.exp                                | 1 +
 44 files changed, 44 insertions(+)

diff --git a/IbaTools/FastFabric/chassis_configure.exp b/IbaTools/FastFabric/chassis_configure.exp
index f18397f..e373a46 100644
--- a/IbaTools/FastFabric/chassis_configure.exp
+++ b/IbaTools/FastFabric/chassis_configure.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmconfig.exp b/IbaTools/FastFabric/chassis_fmconfig.exp
index 986d799..3370e4a 100755
--- a/IbaTools/FastFabric/chassis_fmconfig.exp
+++ b/IbaTools/FastFabric/chassis_fmconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmcontrol.exp b/IbaTools/FastFabric/chassis_fmcontrol.exp
index ea3d546..309aa00 100755
--- a/IbaTools/FastFabric/chassis_fmcontrol.exp
+++ b/IbaTools/FastFabric/chassis_fmcontrol.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmgetconfig.exp b/IbaTools/FastFabric/chassis_fmgetconfig.exp
index dff298e..e3e7ce1 100755
--- a/IbaTools/FastFabric/chassis_fmgetconfig.exp
+++ b/IbaTools/FastFabric/chassis_fmgetconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp b/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp
index 99c38c1..db133dc 100755
--- a/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp
+++ b/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmsecurityfiles.exp b/IbaTools/FastFabric/chassis_fmsecurityfiles.exp
index b45aff1..f94d626 100755
--- a/IbaTools/FastFabric/chassis_fmsecurityfiles.exp
+++ b/IbaTools/FastFabric/chassis_fmsecurityfiles.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_getconfig.exp b/IbaTools/FastFabric/chassis_getconfig.exp
index 7454afc..4d7b265 100644
--- a/IbaTools/FastFabric/chassis_getconfig.exp
+++ b/IbaTools/FastFabric/chassis_getconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_reboot.exp b/IbaTools/FastFabric/chassis_reboot.exp
index f59330f..0574cba 100755
--- a/IbaTools/FastFabric/chassis_reboot.exp
+++ b/IbaTools/FastFabric/chassis_reboot.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_upgrade.exp b/IbaTools/FastFabric/chassis_upgrade.exp
index c255e01..e031313 100755
--- a/IbaTools/FastFabric/chassis_upgrade.exp
+++ b/IbaTools/FastFabric/chassis_upgrade.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/configipoib.exp b/IbaTools/FastFabric/configipoib.exp
index 7918866..ca1a11a 100755
--- a/IbaTools/FastFabric/configipoib.exp
+++ b/IbaTools/FastFabric/configipoib.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/ff_function.exp b/IbaTools/FastFabric/ff_function.exp
index 3590661..3336c7d 100755
--- a/IbaTools/FastFabric/ff_function.exp
+++ b/IbaTools/FastFabric/ff_function.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/ipoibping.exp b/IbaTools/FastFabric/ipoibping.exp
index 17b0486..fc48d02 100755
--- a/IbaTools/FastFabric/ipoibping.exp
+++ b/IbaTools/FastFabric/ipoibping.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/load.exp b/IbaTools/FastFabric/load.exp
index 91991e0..14b0ebf 100755
--- a/IbaTools/FastFabric/load.exp
+++ b/IbaTools/FastFabric/load.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/mpiperf.exp b/IbaTools/FastFabric/mpiperf.exp
index 630dc66..d87ccba 100755
--- a/IbaTools/FastFabric/mpiperf.exp
+++ b/IbaTools/FastFabric/mpiperf.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/mpiperfdeviation.exp b/IbaTools/FastFabric/mpiperfdeviation.exp
index a7fc679..c6dfc2f 100755
--- a/IbaTools/FastFabric/mpiperfdeviation.exp
+++ b/IbaTools/FastFabric/mpiperfdeviation.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/reboot.exp b/IbaTools/FastFabric/reboot.exp
index f27b50a..79a32a0 100755
--- a/IbaTools/FastFabric/reboot.exp
+++ b/IbaTools/FastFabric/reboot.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/sacache.exp b/IbaTools/FastFabric/sacache.exp
index 40a0340..902bd84 100755
--- a/IbaTools/FastFabric/sacache.exp
+++ b/IbaTools/FastFabric/sacache.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_capture.exp b/IbaTools/FastFabric/switch_capture.exp
index ed149a3..03a8fb1 100755
--- a/IbaTools/FastFabric/switch_capture.exp
+++ b/IbaTools/FastFabric/switch_capture.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_configure.exp b/IbaTools/FastFabric/switch_configure.exp
index 3516482..91fa0ba 100644
--- a/IbaTools/FastFabric/switch_configure.exp
+++ b/IbaTools/FastFabric/switch_configure.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_dump.exp b/IbaTools/FastFabric/switch_dump.exp
index b567f26..3560ad2 100755
--- a/IbaTools/FastFabric/switch_dump.exp
+++ b/IbaTools/FastFabric/switch_dump.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_fwverify.exp b/IbaTools/FastFabric/switch_fwverify.exp
index 6be1ea9..c386431 100755
--- a/IbaTools/FastFabric/switch_fwverify.exp
+++ b/IbaTools/FastFabric/switch_fwverify.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_getconfig.exp b/IbaTools/FastFabric/switch_getconfig.exp
index a1cbaae..8c7812f 100755
--- a/IbaTools/FastFabric/switch_getconfig.exp
+++ b/IbaTools/FastFabric/switch_getconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_hwvpd.exp b/IbaTools/FastFabric/switch_hwvpd.exp
index 24fa8a5..e7435fd 100755
--- a/IbaTools/FastFabric/switch_hwvpd.exp
+++ b/IbaTools/FastFabric/switch_hwvpd.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_info.exp b/IbaTools/FastFabric/switch_info.exp
index 73ef67a..74fd00c 100755
--- a/IbaTools/FastFabric/switch_info.exp
+++ b/IbaTools/FastFabric/switch_info.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_ping.exp b/IbaTools/FastFabric/switch_ping.exp
index 04f060e..dfb6bc7 100755
--- a/IbaTools/FastFabric/switch_ping.exp
+++ b/IbaTools/FastFabric/switch_ping.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_reboot.exp b/IbaTools/FastFabric/switch_reboot.exp
index 81f4e3e..ab10142 100755
--- a/IbaTools/FastFabric/switch_reboot.exp
+++ b/IbaTools/FastFabric/switch_reboot.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_upgrade.exp b/IbaTools/FastFabric/switch_upgrade.exp
index 81bd9b4..1f68f37 100755
--- a/IbaTools/FastFabric/switch_upgrade.exp
+++ b/IbaTools/FastFabric/switch_upgrade.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/upgrade.exp b/IbaTools/FastFabric/upgrade.exp
index aecccbb..85b22c2 100755
--- a/IbaTools/FastFabric/upgrade.exp
+++ b/IbaTools/FastFabric/upgrade.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/basic.exp b/TestTools/basic.exp
index 4ffe1f5..b3362ab 100755
--- a/TestTools/basic.exp
+++ b/TestTools/basic.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/chassis.exp b/TestTools/chassis.exp
index 9af2e9a..3ac72db 100755
--- a/TestTools/chassis.exp
+++ b/TestTools/chassis.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/common_funcs.exp b/TestTools/common_funcs.exp
index 68d55d8..7f62eb8 100644
--- a/TestTools/common_funcs.exp
+++ b/TestTools/common_funcs.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/defs.exp b/TestTools/defs.exp
index 8e1a8e6..ecb3e64 100644
--- a/TestTools/defs.exp
+++ b/TestTools/defs.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/extmng.exp b/TestTools/extmng.exp
index 9defebe..1b713eb 100755
--- a/TestTools/extmng.exp
+++ b/TestTools/extmng.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/ib.exp b/TestTools/ib.exp
index 06c4a6e..8e949cd 100755
--- a/TestTools/ib.exp
+++ b/TestTools/ib.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/ibtools.exp b/TestTools/ibtools.exp
index 36b945c..36bfdbf 100644
--- a/TestTools/ibtools.exp
+++ b/TestTools/ibtools.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/install.exp b/TestTools/install.exp
index 7736765..8f64eb3 100755
--- a/TestTools/install.exp
+++ b/TestTools/install.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/mpi.exp b/TestTools/mpi.exp
index ed49434..9180b5b 100755
--- a/TestTools/mpi.exp
+++ b/TestTools/mpi.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/network.exp b/TestTools/network.exp
index a859022..4a81b14 100755
--- a/TestTools/network.exp
+++ b/TestTools/network.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/opa_to_xml.exp b/TestTools/opa_to_xml.exp
index e5e8ab8..77f8311 100644
--- a/TestTools/opa_to_xml.exp
+++ b/TestTools/opa_to_xml.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/proc_mgr.exp b/TestTools/proc_mgr.exp
index abcb65d..94cb60f 100755
--- a/TestTools/proc_mgr.exp
+++ b/TestTools/proc_mgr.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/sm_control.exp b/TestTools/sm_control.exp
index 7f08ccb..bc89ad4 100755
--- a/TestTools/sm_control.exp
+++ b/TestTools/sm_control.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/switch_methods.exp b/TestTools/switch_methods.exp
index 9f13773..3b2b14f 100644
--- a/TestTools/switch_methods.exp
+++ b/TestTools/switch_methods.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/target.exp b/TestTools/target.exp
index 615e87d..1a127a4 100755
--- a/TestTools/target.exp
+++ b/TestTools/target.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/tools.exp b/TestTools/tools.exp
index 850e789..aa7f825 100755
--- a/TestTools/tools.exp
+++ b/TestTools/tools.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
-- 
2.14.0

